src/hotspot/share/gc/shenandoah/shenandoahGenerationSizer.cpp

```diff
@@ -124,16 +124,6 @@ bool ShenandoahGenerationSizer::transfer_regions(ShenandoahGeneration* src, Shen
     return false;
   }
 
-  if (dst->max_capacity() + bytes_to_transfer > max_size_for(dst)) {
```

Comment:
Do we need to edit the descriptions of ShenandoahMinYoungPercentage and ShenandoahMaxYoungPercentage?  Do we need to remove these options entirely from shenandoah_globals?

Reply:
Yes, we should probably remove one of them and name the other `ShenandoahInitYoungPercentage`. I think I will back out this change to `shGenerationSizer` in this PR, and open a different PR for removing this constraint and renaming the options. It's a bit outside the scope of this bug fix.